fastlane/Fastfile

```diff
@@ -372,6 +372,39 @@ end
     comment_body = "You can test the changes on this Pull Request by downloading it from AppCenter [here](#{install_url}) with build number: #{build_number}. IPA is available [here](#{download_url}). If you need access to this, you can ask a maintainer to add you."
     File.write("comment.json", { body: comment_body }.to_json)
   end
+
+  #####################################################################################
+  # Screenshot lanes
+  #####################################################################################
+
+  desc "Runs through the app taking screenshots"
+  lane :take_screenshots do |options|
+    devices = [
+      "iPhone 8 Plus"
+    ]
+
+    languages = [
+      "en-US"
+    ]
+
+    capture_ios_screenshots(
+      scheme: "SimplenoteScreenshots",
+
+      localize_simulator: true,
+      languages: languages,
+
+      devices: devices,
+
+      output_directory: screenshots_directory,
+      clear_previous_screenshots: true,
+
+      # Retry a few times if something is a little flaky
+      number_of_retries: 3,
+
+      # But fail completely after those 3 retries
+      stop_after_first_error: true
+    )
```

Comment:
Not so loosely inspired by https://github.com/woocommerce/woocommerce-ios/blob/1cff1b31fcfa9809c9d852716a529dc20e9294a6/fastlane/Fastfile#L343-L403